src/internal/types.ts

```diff
@@ -1,4 +1,4 @@
-export type DeviceType = 'Handset' | 'Tablet' | 'Tv' | 'Desktop' | 'unknown';
+export type DeviceType = 'Handset' | 'Tablet' | 'Tv' | 'Desktop' | 'GameConsole' | 'unknown';
```

Comment:
```suggestion
export type DeviceType = 'Handset' | 'Tablet' | 'Tv' | 'Desktop' | 'GamingConsole' | 'unknown';
```

Gaming instead of game?

Reply:
That works for me. Updated!